beatrix/src/test/resources/catalogs/testCatalogValidation/CatalogValidation-v2.xml

```diff
@@ -45,7 +45,7 @@
             <product>Standard</product>
             <initialPhases>
             </initialPhases>
-            <finalPhase type="FIXEDTERM">
+            <finalPhase type="EVERGREEN">
```

Comment:
⚠️  Will changing this not affect the test that was written for https://github.com/killbill/killbill/issues/1481 - we need two version of the same Plan, one with `EVERGREEN` and the other with `FIXEDTERM` ?